Source/ChilliSource/Rendering/Texture/TextureDesc.h

```diff
@@ -83,21 +85,6 @@ namespace ChilliSource
         ///
         void SetMipmappingEnabled(bool mipmappingEnabled) noexcept { m_mipmappingEnabled = mipmappingEnabled; };
         
-        /// Sets whether or not the texture data should be restored after a context loss. This involves
-        /// maintaining a copy of the texture data in memory which is costly so this should be disabled
-        /// for any textures that can easily be recreated, i.e any texture that is rendered into every
-        /// frame.
-        ///
-        /// This has no effect on textures that are loaded from file as they are always restored from
-        /// disk.
-        ///
-        /// This will only work for RGBA8888, RGB888, RGBA4444 and RGB565 textures.
-        ///
-        /// @param restoreTextureDataEnabled
-        ///     Whether or not texture data should be restored on context loss.
-        ///
-        void SetTextureDataRestoreEnabled(bool restoreTextureDataEnabled) noexcept;
```

Comment:
Moving this to construction fixed a nasty bug where you could leave the restorable flag defaulted to true and pass in an unsupported image format, the flag was then never checked against the format for compatibility.